eng/pipelines/runtimelab.yml

```diff
@@ -0,0 +1,183 @@
+    buildConfig: release
+    platforms:
+    - Windows_NT_x64
+    jobParameters:
+      testGroup: innerloop
+      nameSuffix: All_Configurations
+      buildArgs: -s clr.corelib+libs -c $(_BuildConfig) -allConfigurations
+
+#
+# CoreCLR Test builds using live libraries release build
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/common/templates/runtimes/build-test-job.yml
+    buildConfig: Checked
+    platforms:
+    - Linux_x64
+    jobParameters:
+      testGroup: innerloop
+      liveLibrariesBuildConfig: Release
+      dependsOn:
+      - build_Linux_x64_Checked_
+      - build_Linux_x64_Release_
+
+#
+# CoreCLR Test executions using live libraries
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/common/templates/runtimes/run-test-job.yml
+    buildConfig: Checked
+    platforms:
+    - Linux_x64
+    helixQueueGroup: pr
+    helixQueuesTemplate: /eng/pipelines/coreclr/templates/helix-queues-setup.yml
+    jobParameters:
+      testGroup: innerloop
+      liveLibrariesBuildConfig: Release
+      dependsOn:
+      - coreclr_common_test_build_p0_AnyOS_AnyCPU_Checked
+
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/common/templates/runtimes/run-test-job.yml
+    buildConfig: Checked
+    platforms:
+    - Windows_NT_x64
+    helixQueueGroup: pr
+    helixQueuesTemplate: /eng/pipelines/coreclr/templates/helix-queues-setup.yml
+    jobParameters:
+      testGroup: innerloop
+      liveLibrariesBuildConfig: Release
+      dependsOn:
+      - coreclr_common_test_build_p0_AnyOS_AnyCPU_Checked
+      - build_Windows_NT_x64_Checked_
+      - build_Windows_NT_x64_Release_
+
+#
+# Libraries Release Test Execution against a release coreclr runtime
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/libraries/run-test-job.yml
+    buildConfig: Release
+    platforms:
+    - Linux_x64
+    helixQueuesTemplate: /eng/pipelines/libraries/helix-queues-setup.yml
+    jobParameters:
+      isFullMatrix: false
+      isOfficialBuild: false
+      testScope: innerloop
+      liveRuntimeBuildConfig: Release
+      dependsOnTestBuildConfiguration: Release
+      dependsOnTestArchitecture: x64
+      dependsOn:
+      - build_Linux_x64_Release_
+
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/libraries/run-test-job.yml
+    buildConfig: Release
+    platforms:
+    - Windows_NT_x64
+    helixQueuesTemplate: /eng/pipelines/libraries/helix-queues-setup.yml
+    jobParameters:
+      isFullMatrix: false
+      isOfficialBuild: false
+      testScope: innerloop
+      liveRuntimeBuildConfig: Release
+      dependsOnTestBuildConfiguration: Release
+      dependsOnTestArchitecture: x64
+      dependsOn:
+      - build_Windows_NT_x64_Release_
```

Comment:
This will add a lot of duplication if we want to build for more platforms where the only thing that differs is the `dependsOn`. I guess we could be smarter later on and have a prefix and then use `osGroup`, `archType`, `buildConfig` and a `suffix` and then construct the dependsOn on the job template directly.

Don't have to address this on the PR but just a note that it might make the entry point yml "cleaner"

Reply:
I agree, I could also just keep a single one but then would have to wait for build that aren't really required in order to start running other jobs.